charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -16,24 +16,48 @@ spike_limit_percentage: 25
 Merge user supplied config into memory limiter config.
 */}}
 {{- define "opentelemetry-collector.baseConfig" -}}
-{{- $processorsConfig := get .Values.config "processors" }}
-{{- if not $processorsConfig.memory_limiter }}
-{{-   $_ := set $processorsConfig "memory_limiter" (include "opentelemetry-collector.memoryLimiter" . | fromYaml) }}
-{{- end }}
+  {{- $processorsConfig := get .Values.config "processors" }}
+  {{- if not $processorsConfig.memory_limiter }}
+  {{-   $_ := set $processorsConfig "memory_limiter" (include "opentelemetry-collector.memoryLimiter" . | fromYaml) }}
+  {{- end }}
 
-{{- if .Values.useGOMEMLIMIT }}
-  {{- if (((.Values.config).service).extensions) }}
-    {{- $_ := set .Values.config.service "extensions" (without .Values.config.service.extensions "memory_ballast") }}
-  {{- end}}
-  {{- $_ := unset (.Values.config.extensions) "memory_ballast" }}
-{{- else }}
-  {{- $memoryBallastConfig := get .Values.config.extensions "memory_ballast" }}
-  {{- if or (not $memoryBallastConfig) (not $memoryBallastConfig.size_in_percentage) }}
-  {{-   $_ := set $memoryBallastConfig "size_in_percentage" 40 }}
+  {{- if .Values.useGOMEMLIMIT }}
+    {{- if (((.Values.config).service).extensions) }}
+      {{- $_ := set .Values.config.service "extensions" (without .Values.config.service.extensions "memory_ballast") }}
+    {{- end}}
+    {{- $_ := unset (.Values.config.extensions) "memory_ballast" }}
+  {{- else }}
+    {{- $memoryBallastConfig := get .Values.config.extensions "memory_ballast" }}
+    {{- if or (not $memoryBallastConfig) (not $memoryBallastConfig.size_in_percentage) }}
+    {{-   $_ := set $memoryBallastConfig "size_in_percentage" 40 }}
+    {{- end }}
   {{- end }}
+
+  {{- .Values.config | toYaml }}
 {{- end }}
 
-{{- .Values.config | toYaml }}
+{{/*
+Minimal collector config for the supervisor preset.
+*/}}
+{{- define "opentelemetry-collector.supervisorCollectorConfig" -}}
```

Comment:
Can we make this configureable? Default enabled when supervisor enabled, but some hidden option to disable this.

Might be useful for us, and also for future